pkg/client/init.go

```diff
@@ -47,6 +50,19 @@ var (
 	metadataClient                 metadata.Interface
 	latestDirectCSIDriveInterface  directcsi.DirectCSIDriveInterface
 	latestDirectCSIVolumeInterface directcsi.DirectCSIVolumeInterface
+
+	binaryName = func() string {
```

Comment:
Can you move these two to the utils package. We shouldn't have the same definitions in two different places.